pandas/tests/io/test_pytables.py

```diff
@@ -4731,6 +4731,34 @@ def test_read_py2_hdf_file_in_py3(self, datapath):
             result = store['p']
             assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("call", [
```

Comment:
it is not necessary to parameterize over the method calls, read_hdf is enough

Reply:
Simplified.